_posts/2018-06-21-vision-for-rust-and-wasm.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Our Vision for Rust and WebAssembly
+---
+
+<meta charset="utf-8" />
+
+**Surgically inserting Rust compiled to WebAssembly should be the best choice
```

Comment:
this is abrupt... we may want to say something to the effect of "rust and webassembly is awesome for x,y,z, but for our 2018 push we are focusing on the goal of:"

Reply:
Yeah a small intro would greatly help before jumping into specific goals

Reply:
Oh I see now that I've read more this was the intro I thought it started off as the goals

Reply:
I too feel it is abrupt. Some small intro will be great.